boards/arm/nrf51_ble400/doc/nrf51_ble400.rst

```diff
@@ -0,0 +1,248 @@
+.. _nrf51_ble400:
+
+Waveshare BLE400
+#################
+
+Overview
+********
+
+BLE400 is a mother board designed for BLE4.0 Bluetooth 2.4G Wireless Module. Used together with Core board.
+
+Supported Features
+==================
+
+Mother board
+------------
+
+- Onboard battery holder, multi power supplies are available
+- All the IOs are accessible for easy expansion
+- Integrates CP2102 for debugging
+- LEDs and user keys, compatible with other official boards, easy to use
+
+Core board
+----------
+
+- Onboard chip: nRF51822
+- Communication distance (open outdoor 1M data rate): 30m
+- Frequency range: 2.4GHz
+- Operating voltage: 2.0V ~ 3.6V
+- Operating temperature: -40℃ ~ 85℃
+- Expansion pinheader: all the I/Os except P0.26 and P0.27
+- Pinheader pitch: 2.00mm
+- Antenna: onboard antenna
+
+Features
+========
+
+- 2.4 GHz multiprotocol RF transceiver
+- ARM® Cortex™-M0 32 bit processor
+- 128 bit AES HW encryption
+- 256kB flash & 32kB RAM
```

Comment:
nRF51822_QFAA is 256KB flash and 16 KB RAM

Reply:
ohh - true